app/components/Login.tsx

```diff
@@ -68,17 +72,19 @@ const RequiredWarning = styled.span`
 `
 
 const Login = () => {
-  const [URI, setURI] = useState('');
+  const [ URI, setURI ] = useState('');
```

Comment:
we should pick one or the other spacing styles `[ s, s1 ]` | `[s, s1]`

Reply:
I like the ones that have the space on the inside. What do you guys think? @AHyman18 @katamhsw @saylestyler

Reply:
i'd say none, not bc i think it looks better but bc it's what react uses in [their docs](https://reactjs.org/docs/hooks-intro.html)

Reply:
that works for me too, ill update 👍